storage/buckets/getUniformBucketLevelAccess.go

```diff
@@ -0,0 +1,53 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+package buckets
+
+// [START storage_get_uniform_bucket_level_access]
+import (
+	"context"
+	"fmt"
+	"io"
+
+	"cloud.google.com/go/storage"
+)
+
+// getUniformBucketLevelAccess gets uniform bucket-level access.
+func getUniformBucketLevelAccess(w io.Writer, bucketName string) (*storage.BucketAttrs, error) {
+	// bucketName := "bucket-name"
+	ctx := context.Background()
+	client, err := storage.NewClient(ctx)
+	if err != nil {
+		return nil, err
+	}
+	defer client.Close()
+
+	attrs, err := client.Bucket(bucketName).Attrs(ctx)
+	if err != nil {
+		return nil, err
+	}
+	uniformBucketLevelAccess := attrs.UniformBucketLevelAccess
+	if uniformBucketLevelAccess.Enabled {
+		fmt.Fprintf(w, "Uniform bucket-level access is enabled for %q.\n",
+			attrs.Name)
+		fmt.Fprintf(w, "Bucket will be locked on %q.\n",
+			uniformBucketLevelAccess.LockedTime)
```

Comment:
This also looks strange? `gofmt`? I think, we should try to write it manually  like:
```
Fprintf(
    w,
    "Uniform ...",
    attrs.Name
)
```
And then run `gofmt` over it

Reply:
Done